### ISSUE TITLE
Pwn Machine Password?

### ISSUE THREAD

**Issue by g0lr** [yeswehack/pwn-machine#12]
Hello, i want to try the pwn-machine, i install the tool, but when i access via IP, i need a password and OTP where i can setup  this?

When you start the PwnMachine for the first time, we ask users to set a new password and 2FA authentication.

But i install 3 times different machines, and doesn't show that

thanks

**Comment by B-i-t-K** [comment]
On the first run pwn-machine should ask you to setup your password and OTP,

If you can't remember it or if something weird happened, you can delete the redis volume to start the installation process again.

**Comment by g0lr** [comment]
Yes, i read that, but when i first go to the IP where the pwn-machine is installed showme only Login page.

I try to install in diferent s.o ubuntu,debian diferent versions, and still the same problem.

Im installing in a VPS this can be the problem?
